### PR TITLE
[doc] Fix copy.Error casing

### PR DESCRIPTION
**Pull request by andresdelfino**

### CODE REVIEW

File: Doc/library/copy.rst [hunk 1]
```diff
@@ -27,7 +27,7 @@ Interface summary:
    Return a deep copy of *x*.
 
 
-.. exception:: error
+.. exception:: Error
```
Comment (tirkarthi) [inline Doc/library/copy.rst:30]: Note: error is an alias for Error although Error is present in `__all__` too

https://github.com/python/cpython/blob/2aabc3200bf03d2ec1aa987e1e20db704948111e/Lib/copy.py#L55-L57

Comment (andresdelfino) [reply Doc/library/copy.rst:30]: @tirkarthi If "error" is a backward compatibility wart, we should be documenting Error instead, right?

Comment (tirkarthi) [reply Doc/library/copy.rst:30]: I was under the impression that with that comment Python 2 only had error and not Error. But the same alias is present in Python 2 too. So I guess it predates something more than 2.7. I agree with the change over documenting Error. I will wait for another core dev opinion. The git blame is as below : 

```
227b1204681 (Fred Drake             2000-08-17 05:06:49 +0000  55) class Error(Exception):
88869f9787c (Tim Peters             2001-01-14 23:36:06 +0000  56)     pass
88869f9787c (Tim Peters             2001-01-14 23:36:06 +0000  57) error = Error   # backward compatibility
```

Comment (orsenthil) [reply Doc/library/copy.rst:30]: Documenting `Error` seems alright and seems more modern usage to me.  +1 with this change.

Comment (orsenthil) [reply Doc/library/copy.rst:30]: Given `Error` is exposed via  `__all__` interface, it is accurate to update the documentation to reflect that.

**Pull request merged**